packages/rocketchat-livechat/server/methods/takeInquiry.js

```diff
@@ -0,0 +1,43 @@
+Meteor.methods({
+	'livechat:takeInquiry'(inquiry, agent) {
+
+		console.log("this");
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2756771086/issues/source?bid=3199634&fileBranchId=3348698#l4)